installer/create-cluster-gke.sh

```diff
@@ -23,6 +23,22 @@ if [ $# -eq 0 ]
     exit 1
 fi
 
+gcloudZone=$(gcloud config get-value compute/zone)
```

Comment:
Maybe we could pass these values as parameters to the installer script and then you can always add `--zone` and `--region` to gcloud command. Then we could have multiple clusters setup independent of the default config per [project id](https://cloud.google.com/compute/docs/gcloud-compute/#default-properties).

Reply:
That would definitely make it more flexible. I can do another PR in which I add 2 mandatory parameters for `--zone` and `--region`. Would we then want to remove the gcloud check for zone and region again and simply restrict the user to always supply the parameters? Alternatively, we could make the 2 parameters optional and fallback to the gcloud configuration